src/frontend/gatsby/src/components/SearchPage/SearchResults.jsx

```diff
@@ -69,7 +76,11 @@ const SearchResults = ({ text, filter }) => {
       {data && data.length ? (
         <Timeline pages={data} nextPage={() => setSize(size + 1)} />
       ) : (
-        <h1>No search results</h1>
+        <div className={classes.noResults}>
+          <img src={NotFound} alt="Not Found" height={200} width={200} />
```

Comment:
@izhuravlev thanks for optimizing the images. Can we be consistent with the image alternates?

Reply:
Alright